packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -47,6 +47,17 @@ enum _RefreshIndicatorMode {
   canceled, // Animating the indicator's fade-out after not arming.
 }
 
+/// Used to configure the [RefreshIndicator] trigger mode.
+enum RefreshIndicatorTriggerMode {
+  /// The indicator can trigger when the child's [Scrollable] descendant overscrolls
+  /// by drag(not inertia), regardless of whether its initial scroll position is on the edge or not.
+  loose,
+
+  /// The indicator can only trigger when the child's [Scrollable] descendant overscrolls
+  /// by drag and its initial scroll position is on the edge.
+  strict,
```

Comment:
strict sounds too generic. we should be specific.

Reply:
Do you have any suggestions for this name？ ：）

Reply:
I am not an expert on this. here is my suggestion.

strict :`onEdge`, `dragStartsAtScrollableEdge`
loose: `anywhere`, `dragStartsAtAnyWhere`